ArduPlane/quadplane.cpp

```diff
@@ -3532,6 +3550,21 @@ void QuadPlane::Log_Write_QControl_Tuning()
  */
 float QuadPlane::forward_throttle_pct()
```

Comment:
Maybe return bool with the throttle and angle limit as pass by reference values.

Reply:
That was the previous interface definition for this function. It doesn't make sense to me to change it to be a combined throttle and pitch angle limit calculator unless we are also getting rid of the legacy method.